nucliadb/src/nucliadb/train/generators/field_classifier.py

```diff
@@ -38,6 +39,7 @@ def field_classification_batch_generator(
     kbid: str,
     trainset: TrainSet,
     shard_replica_id: str,
+    filter_expression: Optional[FilterExpression],
```

Comment:
Maybe thrown an error in all the trainsets where this is not used if it's not none? Something like `filter_expression` not supported for field classifier or whatever. Just to avoid surprises if someone tries to use this.